__tests__/app.test.js

```diff
@@ -59,6 +61,27 @@ describe("2. GET/api/articles/:article_id", () => {
         expect(body.article).toMatchObject(thirdArticle);
       });
   });
+  test("Status:200, responds a selected article that contains comment_count object", () => {
+    const time = "2020-06-06T09:10:00.000Z";
+    const articleID = 9;
+    const ninthArticle = {
+      article_id: 9,
+      title: "They're not exactly dogs, are they?",
+      topic: "mitch",
+      author: "butter_bridge",
+      body: "Well? Think about it.",
+      created_at: time,
+      votes: 0,
+      comment_count: 2,
+    };
+    return request(app)
+      .get(`/api/articles/${articleID}`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual(ninthArticle);
+        expect(body.article.comment_count).toBe(2);
+      });
```

Comment:
We only really needed to test the second assertion here as the earlier test for /articles/article_id covered the other assertion